spanish/wg-diversity-inclusion/focus-areas/project-and-community/README.md

```diff
@@ -1,19 +1,15 @@
-## Project & Communities
+##Proyecto y Comunidades
```

Comment:
I think you need to leave a blank space right after the hashtag.

Reply:
It looked good in the preview, so I think there's no need to change it